e2e/run-programmatically-multiple-projects/run-jest.js

```diff
@@ -0,0 +1,15 @@
+const {runCLI} = require('../../packages/jest/build/jest');
+
+const config = {
+  projects: [
+    {testMatch: ['<rootDir>/client/**/*.test.js']},
+    {testMatch: ['<rootDir>/server/**/*.test.js']},
+  ],
+};
+
+runCLI({config: JSON.stringify(config)}, [process.cwd()])
+  .then(() => console.log('✅ Done'))
```

Comment:
could have something less generic so we're certain it showed up? Or, not certain, but easier for future readers of these tests to understand it's not Jest itself which prints "Done"

Reply:
Good point, changed it to `"run-programmatically-mutiple-projects completed"`